src/Topics/TexasTransfers.hs

```diff
@@ -0,0 +1,167 @@
+                      <~ T.allVulnerabilities <~ [T.South, T.East]
+
+
+completeTransferDoubleton :: Situations
+completeTransferDoubleton = let
+    sit (partnerBid, ourBid, suit) = let
+        action = do
+            setOpener T.South
+            B.b1N
+            makePass
+            _ <- partnerBid  -- Make the linter happy
+            makePass
+            suitLength suit 2
+        explanation =
+            "We have opened " .+ T.Bid 1 T.Notrump .+ ", and partner has " .+
+            "made a Texas Transfer. Complete the transfer, and see what " .+
+            "they do next. Even if we only have a doubleton, partner is " .+
+            "promising a 6-card suit, so we've got a fit."
+        in situation "comp2" action ourBid explanation
+  in
+    -- Same optimization here: don't have North make a Texas Transfer as a
+    -- passed hand.
+    wrap $ return sit <~ [ (B.b1N4D, B.b1N4D4H, T.Hearts)
+                         , (B.b1N4H, B.b1N4H4S, T.Spades)]
+                      <~ T.allVulnerabilities <~ [T.South, T.East]
+
+
+-- WARNING: This situation is rare, and typically requires generating 100,000
+-- boards. Use it sparingly.
+completeTransferSuperfit :: Situations
+completeTransferSuperfit = let
+    sit (partnerBid, ourBid, suit) = let
+        action = do
+            setOpener T.South
+            B.b1N
+            makePass
+            _ <- partnerBid  -- Make the linter happy
+            makePass
+            suitLength suit 5
+        explanation =
+            "We have opened " .+ T.Bid 1 T.Notrump .+ ", and partner has " .+
+            "made a Texas Transfer. Complete the transfer, and see what " .+
+            "they do next. Even if we know we have an 11-card fit, " .+
+            "partner might only have 10 HCP. Leave it up to them whether " .+
+            "to investigate slam."
+        in situation "comp5" action ourBid explanation
+  in
+    -- Same optimization here: don't have North make a Texas Transfer as a
+    -- passed hand.
+    wrap $ return sit <~ [ (B.b1N4D, B.b1N4D4H, T.Hearts)
+                         , (B.b1N4H, B.b1N4H4S, T.Spades)]
+                      <~ T.allVulnerabilities <~ [T.South, T.East]
+
+
+transferSlamInvite :: Situations
+transferSlamInvite = let
+    sit (bid, suit) = let
+        action = do
+            setOpener T.North
+            B.b1N
+            makePass
+            minSuitLength suit 6
+            B.slamInvite
+        explanation =
+            "Partner has opened " .+ T.Bid 1 T.Notrump .+ ", and you've " .+
+            "got a 6-card major and a slam invite. Make a Jacoby transfer, " .+
+            "then raise to game. If partner is a maximum, they'll " .+
+            "investigate slam, and with a minimum, they'll pass."
+        in situation "SInv" action bid explanation
+  in
+    wrap $ return sit <~ [(B.b1N2D, T.Hearts), (B.b1N2H, T.Spades)]
+                      <~ T.allVulnerabilities <~ [T.North, T.West]
```

Comment:
Put in a couple more situations of this from the other side: you open 1N, partner makes a Jacoby transfer and raises to game. With a maximum and a decent fit, bid on, and with a minimum and a doubleton for partner, pass. What does bidding on look like? Probably 4N, I guess?